tests/manage/mcg/test_pv_pool.py

```diff
@@ -169,29 +176,29 @@ def test_pv_scale_out(self, backingstore_factory):
                     pytest.mark.skipif_ocs_version("<4.11"),
                 ],
             ),
-            pytest.param(
-                {
-                    "interface": "CLI",
-                    "backingstore_dict": {
-                        "pv": [
-                            (
-                                3,
-                                MIN_PV_BACKINGSTORE_SIZE_IN_GB,
-                                "ocs-storagecluster-ceph-rbd",
-                                "300m",
-                                "500Mi",
-                                "400m",
-                                "600Mi",
-                            )
-                        ]
-                    },
-                },
-                marks=[
-                    tier2,
-                    pytest.mark.polarion_id("OCS-4643"),
-                    pytest.mark.skipif_ocs_version("<4.12"),
-                ],
-            ),
+            # pytest.param(
```

Comment:
I see the parm code is commented, is it intentional?

Reply:
No this is not intentional, ill make the changes